packages/pages/src/App.js

```diff
@@ -52,6 +77,8 @@ export default provideState({
     entries: [],
     name: "",
     content: "",
+    id: "",
+    isEditting: false,
```

Comment:
You don't need this field, you could simply detect edition based on whether `state.id` is `""`.